libcore/ConnectServerDialog.vala

```diff
@@ -598,12 +609,14 @@ public class PF.ConnectServerDialog : Gtk.Dialog {
         private void remove_needs_attention () {
             if (get_style_context ().has_class (Gtk.STYLE_CLASS_NEEDS_ATTENTION)) {
                 get_style_context ().remove_class (Gtk.STYLE_CLASS_NEEDS_ATTENTION);
+                set_icon_from_icon_name (Gtk.EntryIconPosition.SECONDARY, null);
             }
         }
 
         private void add_needs_attention () {
             if (!get_style_context ().has_class (Gtk.STYLE_CLASS_NEEDS_ATTENTION)) {
                 get_style_context ().add_class (Gtk.STYLE_CLASS_NEEDS_ATTENTION);
+                set_icon_from_icon_name (Gtk.EntryIconPosition.SECONDARY, "dialog-warning");
```

Comment:
This should be -symbolic

Reply:
Done.  Its actually not easy to get these icons to show at the moment because the sensitivity of the connect button enforces entry of  user name and password when these fields are shown.  I think on reflection we should allow blank password entry at least because the password may have been remembered by the system.